<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>omni::ui::MainWindow</class>
 <widget class="QMainWindow" name="omni::ui::MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QFrame" name="frameToolBar">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="toolButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>1</pointsize>
          </font>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../rc/ODM.qrc">
           <normaloff>:/logo/omni1rainbow.png</normaloff>:/logo/omni1rainbow.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>56</width>
           <height>56</height>
          </size>
         </property>
         <property name="popupMode">
          <enum>QToolButton::InstantPopup</enum>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonIconOnly</enum>
         </property>
         <property name="autoRaise">
          <bool>false</bool>
         </property>
         <property name="arrowType">
          <enum>Qt::NoArrow</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnNew">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>New</string>
         </property>
         <property name="iconSize">
          <size>
           <width>54</width>
           <height>54</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnOpen">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Open</string>
         </property>
         <property name="iconSize">
          <size>
           <width>54</width>
           <height>54</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSave">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
         <property name="iconSize">
          <size>
           <width>54</width>
           <height>54</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnEditAsNew">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Edit As New</string>
         </property>
         <property name="iconSize">
          <size>
           <width>54</width>
           <height>54</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnScreenSetup">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Screen Setup</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCanvas">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Canvas</string>
         </property>
         <property name="iconSize">
          <size>
           <width>54</width>
           <height>54</height>
          </size>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPositioning">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Positioning</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnWarp">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Warp</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnBlend">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Blend</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnExport">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Export</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="horizontalFrame">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QWidget" name="sideBar" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="grpSettings">
            <property name="title">
             <string>Settings</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="grpInputs">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="title">
             <string>Inputs</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_13">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>6</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>6</number>
             </property>
             <item>
              <widget class="QFrame" name="frameAddRemoveImage">
               <layout class="QHBoxLayout" name="horizontalLayout_15">
                <property name="leftMargin">
                 <number>2</number>
                </property>
                <property name="topMargin">
                 <number>2</number>
                </property>
                <property name="rightMargin">
                 <number>2</number>
                </property>
                <property name="bottomMargin">
                 <number>2</number>
                </property>
                <item>
                 <widget class="QPushButton" name="btnAddInput">
                  <property name="text">
                   <string>Add</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btnRemoveInput">
                  <property name="text">
                   <string>Remove</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QTreeView" name="inputList"/>
             </item>
             <item>
              <widget class="QWidget" name="rotationWidget" native="true"/>
             </item>
             <item>
              <widget class="QGroupBox" name="grpInputOptions">
               <property name="title">
                <string>Input Options</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="startScreen" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>4</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="grpProjectors">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Projectors</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frameTunings">
            <layout class="QHBoxLayout" name="horiz">
             <item>
              <widget class="QPushButton" name="btnAddTuning">
               <property name="text">
                <string>Add</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnRemoveTuning">
               <property name="text">
                <string>Remove</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="omni::ui::proj::TuningList" name="tuningList" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>omni::ui::proj::TuningList</class>
   <extends>QWidget</extends>
   <header location="global">omni/ui/proj/TuningList.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../rc/ODM.qrc"/>
 </resources>
 <connections/>
</ui>
